include/executor.hpp

```diff
@@ -149,6 +149,18 @@ class Executor
         latestSrcAndHexwords = pelEventId;
     }
 
+    /**
+     * @brief API to execute functions on requests from d-bus
+     * This method is called whenever there is an ExecuteFunction request from
```

Comment:
JInu: rephrase: This method is called whenever there is an external request to trigger a function.

Reply:
done